podium-gateway/package.json

```
@@ -92,8 +92,8 @@
     "ts-node": "3.3.0",
     "tslint": "5.5.0",
     "tslint-loader": "3.5.3",
-    "typescript": "2.6.2",
-    "webpack": "3.6.0",
+    "typescript": "4.1.5",
+    "webpack": "4.46.0",
```

Looks like a dependency on `webpack-cli` now needs to be added:
```suggestion
    "webpack": "4.46.0",
    "webpack-cli": "^4.6.0",
```